docs/hive-integration/agent-labels.md

```diff
@@ -0,0 +1,15 @@
+# Agent Inventory Labels
+
+The operator automatically adds labels to each Agent CR based on its inventory.  The goal of these labels is to allow for easily filtering Agents based on these properties.
+
+These labels describe the Agent inventory and not any higher-level usability.  For example, there are labels indicating that virtualization is enabled on the CPU (VMX or SVM CPU flags), but no label indicating that the host is suitable for running CNV.
+
+An annotation on the Agent CR indicates a version for the labels, so clients can anticipate what labels will be applied ("feature.agent-install.openshift.io/version").
+
+## Label list
```

Comment:
It would be useful to describe some of these values in more detail. Booleans will presumably be the lowercase string "true" or "false"? CPU architecture examples would be a good idea, or some other info about what to anticipate: will it be x86-64, x86_64, amd64, AMD64, etc. ?

Reply:
Sure, fixed

Reply:
Sure, fixed